test/extended/builds/pipeline.go

```diff
@@ -13,36 +16,98 @@ import (
 	"github.com/openshift/origin/test/extended/util/jenkins"
 )
 
+func debugAnyJenkinsFailure(br *exutil.BuildResult, name string, oc *exutil.CLI, dumpMaster bool) {
+	if !br.BuildSuccess {
```

Comment:
this seems slightly weird.  Why not just remove the br argument and only call debugAnyJenkinsFailure when desired(namely when the build fails)?